go.mod

```diff
@@ -0,0 +1,6 @@
+module github.com/go-chi/chi/v3
+
+require (
+	golang.org/x/net v0.0.0-20180611182652-db08ff08e862
+	golang.org/x/text v0.3.0
```

Comment:
Where do we use golang.org/x/text ?

Reply:
It's used in `golang.org/x/net` undercover, for Unicode support.
`golang.org/x/net` haven't adopted go.mod yet, so `golang.org/x/text` resolving appears here.

As soon as go.mod is introduced in `golang.org/x/net` and you update it, transitive dependency will go away automatically.

Reply:
Thanks, that sounds good to me.